src/components/Trade/hooks/useTradeAmounts.tsx

```diff
@@ -158,24 +155,22 @@ export const useTradeAmounts = () => {
           break
       }
 
-      const buyAssetIdToUse = buyAssetId ?? buyAssetFormState?.assetId
-      const sellAssetIdToUse = sellAssetId ?? sellAssetFormState?.assetId
-      const amountToUse = amount ?? amountFormState
-      const actionToUse = action ?? actionFormState ?? TradeAmountInputField.SELL_CRYPTO
+      const buyAssetIdToUse = buyAssetId ?? buyAsset?.assetId
+      const sellAssetIdToUse = sellAssetId ?? sellAsset?.assetId
```

Comment:
Can we keep the `state` vernacular here 🙏🏽 ? This is now getting tricky to grasp without going to the definition of `buyAsset`

Reply:
Very reasonable! https://github.com/shapeshift/web/pull/4080/commits/614e917554e262e4415e853bcce6661b436f6587